src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanics.cpp

```diff
@@ -213,9 +213,6 @@ void MultiphasePoromechanics::assembleSystem( real64 const GEOS_UNUSED_PARAM( ti
 
   solidMechanicsSolver()->getMaxForce() = LvArray::math::max( mechanicsMaxForce, poromechanicsMaxForce );
 
-  // tell the flow solver that this is a stress initialization step
-  flowSolver()->keepFlowVariablesConstantDuringInitStep( m_performStressInitialization );
```

Comment:
so this you just moved to a more appropriate place, right? not sure i follow the logic here...